<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>544</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>462</width>
    <height>544</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>462</width>
    <height>544</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ÖTS - Giriş</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/images/images/yu_Icon2.png</normaloff>:/images/images/yu_Icon2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Login{
background:url(:/images/images/ballon.jpg);

}

QFrame{
background:rgba(0,0,0,0.7);
border-radius:15px;
}

QFrame:hover{
background:rgba(0,0,0,0.73);
}

QLineEdit{
font-size:25px;;
color:white;
border:none;
background:transparent;
border-bottom:1px solid white;
}

QLineEdit:hover{
background:rgba(0,0,0,0.04);
}


QPushButton{
background:rgb(83, 152, 255);
border-radius:10px;
color:#fff;
}

QPushButton:hover{
border:none;
background:rgb(68, 117, 222)
}

QCheckBox{
color:silver;
background:transparent
}
QCheckBox:hover{
color:white;
}


</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>80</y>
     <width>420</width>
     <height>440</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>161</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>24</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>Kullanıcı Adı</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>241</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Şifre</string>
    </property>
   </widget>
   <widget class="QLabel" name="baslik">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:40px;
color:white;
background:transparent;</string>
    </property>
    <property name="text">
     <string>Öğrenci Takip Sistemi</string>
    </property>
   </widget>
   <widget class="QLabel" name="baslik_2">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>45</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:18px;
color:white;
background:transparent;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>Y A L O V A   Ü N İ V E R S İ T E S İ</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QCheckBox" name="remember_me">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>318</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Beni Hatırla</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>1</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sign">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>353</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Giriş Yap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_newAccount">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>318</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent
}

QPushButton{
color:silver;
background:transparent
}
QPushButton:hover{
color:white;
}

</string>
    </property>
    <property name="text">
     <string>Yeni Hesap Oluştur</string>
    </property>
   </widget>
   <zorder>baslik</zorder>
   <zorder>username</zorder>
   <zorder>password</zorder>
   <zorder>baslik_2</zorder>
   <zorder>remember_me</zorder>
   <zorder>line</zorder>
   <zorder>btn_sign</zorder>
   <zorder>btn_newAccount</zorder>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>43</y>
     <width>71</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/images/yu_ana_logo2.png&quot; width=&quot;67&quot; height=&quot;105&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>0</y>
     <width>471</width>
     <height>38</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background:rgba(0,0,0,0.38);
}
QFrame{
border-radius:none;
}

QFrame:hover{
background:rgba(0,0,0,0.40);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>3</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:images/images/yu_ana_logo2.png&quot; width=&quot;21&quot; height=&quot;30&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>12</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color:silver;
background:transparent;</string>
    </property>
    <property name="text">
     <string>Giriş Yap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>424</x>
      <y>0</y>
      <width>41</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgba(0, 0, 0,0.15);
color:silver;

border-radius:none;
}


QPushButton:hover{
background:red;
color:white
}</string>
    </property>
    <property name="text">
     <string>✕</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_minimize">
    <property name="geometry">
     <rect>
      <x>383</x>
      <y>0</y>
      <width>41</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgba(0, 0, 0,0.15);
border-radius:none;
}


QPushButton:hover{
background:rgba(0, 0, 0,0.6)
}</string>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>label</zorder>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
